shraga_common/app/services/history_service.py

```diff
@@ -16,6 +16,92 @@
 
 logger = logging.getLogger(__name__)
 
+async def get_chat_dialogs(
```

Comment:
As far as I understand this function is specific to analytics?
Maybe it should be in an analytics service?